config/v201/component_schemas/standardized/InternalCtrlr.json

```
@@ -537,6 +537,22 @@
           "description": "List of criteria supported for a get custom report. Enabled,Active,Problem,Available",
           "default": "Enabled,Active,Problem,Available",
           "type": "string"
+      },
+      "RoundClockAlignedTimestamps": {
+          "variable_name": "RoundClockAlignedTimestamps",
+          "characteristics": {
+              "supportsMonitoring": false,
+              "dataType": "boolean"
+          },
+          "attributes": [
+              {
+                  "type": "Actual",
+                  "mutability": "ReadOnly"
+              }
+          ],
+          "description": "If enabled the metervalues configured with the AlignedDataCtrlr will be rounded to the exact time intervals",
+          "default": "0",
```

```suggestion
          "default": true,
```
I just noticed that this is an issue at several other places

---

Yep that's what made me unsure if this was right. Shall I change those as well?

Also, I thought to leave it off by default or would you rather have it enabled?

---

Yes, would be great if you can change all of them, thanks :+1: . Leave it off by default is fine for me